lunes_cms/help/templates/public_upload.html

```diff
@@ -105,11 +122,11 @@ <h3 class="h5 mb-4 mt-4 font-weight-normal">
       </h3>
       {% if upload_success %}<div class="alert alert-success" role="alert">Vielen Dank für dein Bild. Das Lunes Team wird es prüfen und freigeben.</div>{% endif %}
       <label for="inputDiscipline" class="sr-only">Beruf auswählen</label>
-      <select id="inputDiscipline" class="form-control" required autofocus><option>Bitte Beruf wählen</option></select>
+      <select id="inputDiscipline" class="form-control" required autofocus><option>Bitte Beruf wählen</option><option value="0">Alle Berufe</option></select>
```

Comment:
```suggestion
      <select id="inputDiscipline" class="form-control" required autofocus>
          <option value="none">Bitte Beruf wählen</option>
          <option value="all">Alle Berufe</option>
      </select>
```
To differ between "all disciplines" and "no disciplines", I'd use more clear values than the `0`...